src/model/kaniCommandCreate.ts

```diff
@@ -94,12 +70,12 @@ export async function runCargoKaniTest(
 	const crateURI = getRootDir();
 	let harnessCommand = '';
 	if (args === undefined || NaN) {
-		harnessCommand = `cd ${crateURI} && ${KaniConstants.CargoKaniExecutableName} ${KaniArguments.testsFlag} ${KaniArguments.harnessFlag} ${harnessName}`;
+		harnessCommand = `${KaniConstants.CargoKaniExecutableName} ${KaniArguments.testsFlag} ${KaniArguments.harnessFlag} ${harnessName}`;
```

Comment:
One comment I agree with is that this command should be an array. All arguments could then be pushed into the array. That'd be similar to how we handle commands in Kani with the `Command` type, and removes the need to trim the string.

I'm not going to request this work be done here, but please open an issue and mention it in the PR.

Reply:
Can do, thanks for the suggestion.